cmd/livepeer/livepeer.go

```diff
@@ -227,3 +228,16 @@ func updateNilsForUnsetFlags(cfg starter.LivepeerConfig) starter.LivepeerConfig
 
 	return res
 }
+
+// Defaults session limit to "0" (auto) for orchestrators when -maxSessions flag not set
+func defaultOrchestorLimit(cfg starter.LivepeerConfig) starter.LivepeerConfig {
+	res := cfg
+
+	isFlagSet := make(map[string]bool)
+	flag.Visit(func(f *flag.Flag) { isFlagSet[f.Name] = true })
+	if isFlagSet["orchestrator"] && !isFlagSet["maxSessions"] {
+		*cfg.MaxSessions = 0
```

Comment:
What happens in the case of a standalone O? Won't it now stop accepting any transcoding by default (since its max sessions is set to 0)?

Reply:
Good catch, I tested and confirmed this was an issue. I have removed the default auto session limit.